apps/client/src/components/Location/LocationInput.tsx

```diff
@@ -61,6 +71,14 @@ const LocationInput: React.FC<{
         action: actions.SUBMIT_LOCATION,
         name: address.postalCode.substring(0, 4),
       });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: monument ? monument : eventNames.NO_MONUMENT,
```

Comment:
Can you use the `||` operator here (and below)?

Reply:
Fixed